client/package.json

```diff
@@ -34,15 +34,14 @@
       },
       "requireConfigFile": false
     },
-    "plugins": [
-      "prettier"
-    ],
     "extends": [
       "airbnb",
       "airbnb/hooks",
-      "plugin:prettier/recommended"
+      "../package.json"
     ],
     "rules": {
+      "no-unused-vars": "warn",
+      "no-shadow": "off",
```

Comment:
I don't like turning off no-shadow.

Reply:
So you prefer different variable names for each variables on a single file? alright no worries!